pkg/sloop/store/untyped/db_utilities.go

```diff
@@ -0,0 +1,56 @@
+package untyped
+
+import (
+	"github.com/dgraph-io/badger/v2"
+	"github.com/salesforce/sloop/pkg/sloop/store/untyped/badgerwrap"
+)
+
+var collectSize = 100000
```

Comment:
Update transaction tracks the rows that are touched and hence very large transactions can end up too much of an overhead.
I picked the 100k value based on how other projects on github are using badger update. There is still a chance that we see latency issues arising from it and I plan to monitor and iterate. And hence it may make sense to move this to a config.